ocamltest/builtin_actions.ml

```diff
@@ -217,6 +232,19 @@ let use_runtime backend ocamlsrcdir = match backend with
     "-use-runtime " ^ ocamlrun
   | _ -> ""
 
+let runtime_variant_flags backend ocamlsrcdir =
+  let variant = runtime_variant() in
+  if variant=Normal then ""
+  else begin
+    let variant_str = if variant=Debug then "d" else "i" in
+    let backend_lib = match backend with
+      | Sys.Bytecode -> "byterun"
+      | Sys.Native -> "asmrun"
+      | Sys.Other _ -> "stdlib" in
```

Comment:
Apologies for being very ignorant of runtime variants, but why is `stdlib` required in the `Sys.Other _` backend case?